cucumber/authenticators_azure/features/support/authn_azure_helper.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 # Utility methods for Azure authenticator
+require 'cucumber/_authenticators_common/features/support/authenticator_helpers'
```

Comment:
I guess that this and similar change in other files should be done some other way
For example by editing `cucumber.yml` file

Reply:
It is not clear to me yet why it's necessary.
cucumber.yml already includes all files in this .../support directory.

Also happens in:
`authn_gcp_helper.rb`
`authn_ldap_helper.rb`
`authn_oidc_helper.rb`

I tried to cancel this require line and it failed:
https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur/detail/ruby-3/79/pipeline/209